src/chocolatey.resources/helpers/functions/Get-ChocolateyUnzip.ps1

```diff
@@ -196,6 +197,31 @@ param(
     default { throw "7-Zip signalled an unknown error (code $exitCode)" }
   }
 
-  $env:ChocolateyPackageInstallLocation = $destination
-  return $destination
+  if ((test-path variable:url) -AND (@($url).count -gt 0))
+  {
+    # If our unzip resulted in a .tar file that matches our input zip file without it's extensions, unzip it
+    # does not work with archive names that contain a '.' before the extensions
+    $matchingtarname = "$destinationNoRedirection\$((split-path -leaf $url).split('.')[0]).tar"

```

Comment:
Tar name doesn't need to match, just search for a .tar file in the extracted directory.